src/com/oltpbenchmark/WorkloadConfiguration.java

```diff
@@ -377,10 +317,6 @@ public boolean getUseThinkTime() {
     return useThinkTime;
   }
 
-  public void setUseThinkTime(boolean useThinkTime) {
```

Comment:
is this not used?

Reply:
correct

https://github.com/yugabyte/tpcc/search?q=setUseThinkTime